code/Example/Drasil/GlassBR/Body.hs

```diff
@@ -575,8 +576,13 @@ s7_1_req6 = [(Enumeration $ Simple $ [(acroR 6, Nested (titleize output_ +:+
 
 {--LIKELY CHANGES--}
 
-s8_list :: [Contents]
-s8_list = likelyChanges_SRS 
+s8a_list :: [Contents]
```

Comment:
What does the 8 stand for?  Section 8?  If so, that isn't a great name.  All of the sections can move over time and the numbering is likely to change.  What about sLC_list and sUC_list?  I see LC and UC used elsewhere in the naming; I assume to mean Likely Changes and Unlikely Changes.

Reply:
Yes and yes - should we make an issue for more descriptive section names?

Reply:
An issue sounds like a good idea to me.